pyaedt/modeler/cad/component_array.py

```diff
@@ -0,0 +1,843 @@
+from __future__ import absolute_import
+
+from collections import OrderedDict
+import os
+import re
+
+from pyaedt import pyaedt_function_handler
+from pyaedt.generic.general_methods import _uname
+from pyaedt.generic.general_methods import read_csv
+
+
+class ComponentArray(object):
+    """Manages object attributes for 3DComponent array.
+
+    Parameters
+    ----------
+    app : :class:`pyaedt.Hfss`
+        Hfss pyaedt object.
+    name : str, optional
+        Array name. The default value is ``None``.
+    props : dict, optional
+        Dictionary of properties. The default value is ``None``.
```

Comment:
```suggestion
        Dictionary of properties. The default is ``None``.
```
What behavior occurs when None is used? "The default is None, in which case ...."